### DISCUSSION TITLE
Errore nella creazione di automazione

### DISCUSSION THREAD

**Discussion by effe-7** [indomus/forum#2685] [HUB: Home Assistant]
Sono alle prese nella creazione delle prime automazioni e mi sono subito bloccato..

Sto provando con una semplice con dispositivo z-wave nello specifico un roller shutter 3 fibaro

Attivazione .. quando il sole tramonta
Azione .. chiudere la tapparell

Ma ricevo errore al salvataggio!

**_IMMAGINE RIMOSSA_**

**Comment by snakuzzo** [comment]
Elimino subito.
Non postare foto allo schermo...al massimo fai uno screenshot

**Comment by effe-7** [comment]
Scusa stavo provando a fare interfaccia da pc mentre per github faccio da telefono

**TESTO RIMOSSO**

**Reply by snakuzzo** [reply]
e cortesemente non fare più reply direttamente dalla mail. Grazie.
Rimuovo il testo dall'ultimo messaggio

**Comment by effe-7** [comment]
allego screeshot
![Screenshot 2023-11-03 113702](https://github.com/indomus/forum/assets/83612865/c13ccf80-7b63-4276-9783-0195fff7e8b1)

**Reply by snakuzzo** [reply]
Premesso che da GUI le automazioni sono una m...da e ti converrebbe vederle e farle in YAML, i campi indicati con l'asterisco devono essere popolati. Con cosa devi saperlo tu

**Reply by effe-7** [reply]
ancora scusa effettivamente tramite YAML è stato più comprensivo.
